client/src/components/GraphViewerComponent/GraphViewerRelationshipCreateComponent/GraphViewerRelationshipCreateComponent.js

```diff
@@ -164,16 +163,22 @@ export class GraphViewerRelationshipCreateComponent extends Component {
               errorMessage={hasRequiredRelError ? "Please select a relationship" : null}
               onChange={this.onSelectedRelChange} />
           </div>
-        } {
-          !hasRelationships && <div className="warning-icon-wrapper">
-            <div className="warning-icon">
-              <IconButton iconOnly="true" iconProps={{ iconName: warningIconName }} title="Warning Relationship" ariaLabel="Warning  Relationship" />
-            </div>
-            <div className="warning-text">
-              <Label className="warning-label">No relationship available, try swapping source and target</Label>
-            </div>
-          </div>
         }
+        { !hasRelationships && <MessageBar
+          aria-label="Warning  Relationship"
```

Comment:
Remove extra space

Reply:
I already commit all the Required Changes for this to use translation sorry for the inconvenience had no idea we use it that way.